core/README.md

```diff
@@ -128,3 +180,108 @@ the [overview](../overview.md#example-landing-page) document.
 ## Extensions
 
 None.
+
+## Browseable Catalogs
```

Comment:
This is a really good description.  I was having trouble groking the concept of "browseable" vs. "searchable" in the various issues, but this makes it really clear.